static/app/components/core/switch/index.tsx

```diff
@@ -45,75 +49,85 @@ const SwitchWrapper = styled('div')`
   justify-content: flex-start;
 `;
 
-const NativeHiddenCheckbox = styled('input')<{
-  nativeSize: NonNullable<SwitchProps['size']>;
-}>`
-  position: absolute;
-  opacity: 0;
-  top: 0;
-  left: 0;
-  height: 100%;
-  width: 100%;
-  margin: 0;
-  padding: 0;
-  cursor: pointer;
+const NativeHiddenCheckbox = withChonk(
+  styled('input')<{
+    nativeSize: NonNullable<SwitchProps['size']>;
+  }>`
+    position: absolute;
+    opacity: 0;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    cursor: pointer;
 
-  & + div {
-    > div {
-      background: ${p => p.theme.border};
-      transform: translateX(${p => ToggleConfig[p.nativeSize].top}px);
+    & + div {
+      > div {
+        background: ${p => p.theme.border};
+        transform: translateX(${p => ToggleConfig[p.nativeSize].top}px);
```

Comment:
`ToggleConfig` -> `toggleConfig`
`ToggleWrapperSize` -> `toggleWrapperSize`

please

Reply:
3rd time I owe you a beer 😂 Idk why I did this

Reply:
this is enforceable with eslint so yeah, we shouldn’t need to have this in reviews 😂